components/footer.py

```diff
@@ -45,64 +49,73 @@ def my_footer():
+                                        alt="github logo",
+                                    ),
+                                    href="#",
                                 ),
-                                href="#",
-                            )
-                        ),
-                        span={"base": 6, "md": 1},
-                    ),
-                    dmc.GridCol(
-                        dmc.Center(
-                            dmc.Anchor(
-                                dmc.Image(
-                                    src="/assets/media/linkedin-white.png",
-                                    maw=45,
-                                    alt="linkedin logo",
+                                dmc.Anchor(
+                                    dmc.Image(
+                                        src="/assets/media/linkedin-white.png",
+                                        maw=45,
+                                        alt="linkedin logo",
+                                    ),
+                                    href="#",
                                 ),
-                                href="#",
-                            )
+                            ],
+                            align="center",
+                            justify="center",
+                            gap="xl",
                         ),
-                        span={"base": 6, "md": 1},
+                        span={"base": 12, "md": 2},
                     ),
                     dmc.GridCol(
-                        children=[
-                            dmc.Text(
-                                TextFooter.cite_strong.value,
-                                fw=700,
-                                c="white",
-                                size="sm",
-                            ),
-                            dmc.Text(
-                                TextFooter.cite.value,
-                                c="white",
-                                size="sm",
-                            ),
-                            dmc.Anchor(
+                        dmc.Stack(
+                            [
                                 dmc.Text(
-                                    TextFooter.cite_link.value,
+                                    TextFooter.cite_strong.value,
+                                    fw=700,
                                     c="white",
                                     size="sm",
-                                    td="underline",
                                 ),
-                                href=TextFooter.cite_link.value,
-                            ),
-                        ],
-                        p="1rem",
+                                dmc.Text(
+                                    TextFooter.cite.value,
+                                    c="white",
+                                    size="sm",
+                                ),
+                                dmc.Anchor(
```

Comment:
this text should not be in a new line, but it should be following the previous text

Reply:
ok, I will fix this part

Reply:
Its' now following the previous text
![image](https://github.com/user-attachments/assets/da80cc92-ee6e-4857-85db-c29bb0bda43c)